src/tools/perf/lib/ucp_tests.cc

```diff
@@ -23,7 +23,7 @@ class ucp_perf_test_runner {
 public:
     typedef uint8_t psn_t;
 
-    static const unsigned AM_ID     = 1;
+    static const unsigned AM_ID     = UCP_PERF_DAEMON_AM_ID_PEER_TX;
```

Comment:
it's a bit weird because daemon is not relevant in this flow

Reply:
Yes, naming is a bit confusing here..
If I get it right, Yossi's proposal was to use the same AM ID both for DPU-to-DPU communication and for host-to-host one. So the easiest option I found is just to remove the `DAEMON` part from that enum ID, so that it is now: `UCP_PERF_AM_ID_PEER_TX`
Please let me know if it fits